### ISSUE TITLE
proposal amendments

### ISSUE THREAD

**Issue by orbeckst** [ASU-CompMethodsPhysics-PHY494/final-2017-openthepodbaydoors_hal#1]
@ASU-CompMethodsPhysics-PHY494/openthepodbaydoors_hal – interesting proposal, I am curious to see the results!

1. Put *all* team members on the proposal. Everyone needs to agree on what's in there.
2. The objectives are not clear how you will determine the initial traffic density or if you propose to study traffic flow as a function of density – this would be a good goal O3. Then make current O3 the new O4.
2. I would be fine with making current O4 "Assign destinations" a stretch goal; *definitely* if you make the density dependency a new objective.
3. Comment on what computational approach you are going to use, e.g, [agent based modelling](https://en.wikipedia.org/wiki/Agent-based_model) with an object-oriented code design?

Comment: instead of reflecting boundary conditions on your streets you could also use [periodic boundary conditions](https://en.wikipedia.org/wiki/Periodic_boundary_conditions) (exit N, enter S, exit E, enter W).

**Comment by orbeckst** [comment]
@Pazookii @AZAkyla please update your proposal and leave a message here commenting on how you amended. I will review and then hopefully approve and close the issue.

**Comment by orbeckst** [comment]
Please note: ask questions here and then let me know when you updated the proposal.

**Comment by Pazookii** [comment]
After further discussion with the group and taking the issue tracker notes into consideration, the updated proposal has been pushed. Please advise.

**Comment by orbeckst** [comment]
I re-opened the issue. I am now reviewing. The issue will be closed once the proposal is approved.

**Comment by orbeckst** [comment]
Looking good. Go ahead.

Approved.
